src/main/java/Main.java

```diff
@@ -100,23 +103,53 @@ public static int CrudOption() {
 
     }
 
-    public static void CreatePatient(){
+    public static void createPatient(PatientClass patient,String DatabaseName){
+        PatientClass pat = new PatientClass();
+        System.out.println("Enter ID: ");
+        int ID = myInput.nextInt();
+        System.out.println("Enter Name: ");
+        String name = myInput.next();
+        System.out.println("Enter DOB: ");
+        String dateOfBirthday = myInput.next();
+        System.out.println("Enter Treatment Date: ");
+        String dateOfTreatment= myInput.next();
+        System.out.println("Enter Address: ");
+        String address = myInput.next();
+        System.out.println("Enter Age: ");
+        int age = myInput.nextInt();
+        System.out.println("Enter Allergies: ");
+        String allergies = myInput.next();
+        System.out.println("Is Special Needs?: ");
+        boolean needspecialNeeds = myInput.nextBoolean();
+        System.out.println("Enter Treatment Type: ");
+        String typeOfTreatment = myInput.next();
+        pat.setID(ID);
+        pat.setName(name);
+        pat.setDateOfBirthday(dateOfBirthday);
+        pat.setDateOfTreatment(dateOfTreatment);
+        pat.setAddress(address);
+        pat.setAge(age);
+        pat.setNeedspecialNeeds(needspecialNeeds);
+        pat.setTypeOfTreatment(typeOfTreatment);
+        implement.createPatient(pat);
 
     }
-    public static void showAllPatient(){
-
+    public static void showAllPatient(String DatabaseName){
+        implement.showAllPatient();
     }
 
-    public static void showPatientById(){
-
+    public static void showPatientById(int id, String DatabaseName){
```

Comment:
```suggestion
    public static void showPatientById( String DatabaseName){
```

Why are you asking for an Id to make the class work but in the method you ask the user for the id?
Remember what inside the () is what you need for the class to work but you don't need the id fpr the class to work since inside the class you are going to ask for the id anyways